src/forms/TopComponent/Views/DesktopView.tsx

```diff
@@ -22,7 +22,7 @@ const BIG_SCREEN_BREAKPOINT = BREAKPOINTS.topComponent.bigScreenView + "px";
 
 const DesktopViewColumn = styled(StyledColumn)`
 	justify-content: space-between;
-	padding: ${pr => pr.sections ? "24px 20px 0px 20px" : "24px 20px 20px 20px"};
+	padding: ${pr => pr.sections ? "24px 24px 0px 24px" : "24px 24px 16px 24px"};

```

Comment:
Now that almost all variants will share the same sizes, spaces, and styling, I'm confident we could move all of this code into "StyledColumn" and delete "DesktopViewColumn"